### PR TITLE
Testing Mega Detector

### LINKED ISSUE #19: Testing MegaDetector

### PR DESCRIPTION
**Pull request by jdrockmael**

### CODE REVIEW

File: cnnvgg16.py [hunk 1]
```diff
@@ -21,7 +21,7 @@
 
 # the number of subdirectories within the "Training Data" directory
 numSubdirectories = len(list(os.walk(trainDIR)))
-
+
```
Comment (HoneyJar-0) [inline cnnvgg16.py:24]: This is no change to actual code. no problem adding it

**Review by HoneyJar-0 — APPROVED** [review]
The only change was deleting an empty line and replacing it with a ' ' character. All good

**Comment by liamhall547** [comment]
I've made more changes. Please review.

**Comment by jdrockmael** [comment]
New branches look good

**Pull request merged**